androbot/handlers.py

```diff
@@ -1,24 +1,30 @@
 from aiogram import types as aiotypes
 
-from . import backend, states, views
-from .errors import UserExistsError
+from . import states, views
+from .actions import Actions
+from .config import settings
+from .errors import UserExistsException
 from .main import bot, dp
+from .schemas import TelegramUser
 
 
 @dp.message_handler(commands=["start"], state="*")
 async def send_start_screen(message: aiotypes.Message):
     """
     Обработчик для команды start.
-
-    Выполняет действия:
-    1. Созадем пользователя в базе (если его еще нет)
-    2. Регистрируем событие что пользователь нажал старт
-    3. Формируем ответное сообщение и показываем главное меню
     """
-    backend.register_action("start", message)
+    full_user_name = " ".join(
+        [name for name in [message.from_user.first_name, message.from_user.last_name] if name]
+    )
+    tg_user = TelegramUser(
+        tg_user_id=message.from_user.id,
+        name=full_user_name,
+        username=message.from_user.username,
+        specialty="Android Developer",  # TODO это не должно быть обязательным полем
+    )
 
     try:
-        backend.add_user(message)
+        Actions().add_user(tg_user)
```

Comment:
Нене, давай не так
Я до сюда дочитал и вернулся к месту, где ты описываешь `Actions`

```suggestion
        with Actions() as act:
            act.add_user(tg_user)
```